pkg/node/manager/manager.go

```diff
@@ -362,16 +362,6 @@ func (m *manager) backgroundSync(ctx context.Context) error {
 // legacyNodeIpBehavior returns true if the agent is still running in legacy
 // mode regarding node IPs
 func (m *manager) legacyNodeIpBehavior() bool {
-	// Cilium < 1.7 only exposed the Cilium internalIP to the ipcache
```

Comment:
This file also reads `RemoteNodeIdentitiesEnabled` a bit below in `nodeIdentityLabels`. That usage also needs to be removed.